packages/vega/src/VegaPlot.js

```diff
@@ -24,9 +28,59 @@ export function VegaPlot(props) {
   const {
     spec: partialSpec,
     data,
+    tooltipTextFunc,
```

Comment:
that's not the best name in the world -- happy to hear any better proposals

Reply:
Maybe `formatTooltipText` or `createTooltipText`?  Maybe something that describes its argument?